graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -50,6 +52,8 @@ def _has_any_absolute(selectivities):
     return False
 
 
+# TODO(bojanserafimov): The class name should be checked against the class name of the index.
+# TODO(bojanserafimov): This is not correct for len(filter_fields) > 1.
```

Comment:
Will address this in a future PR. In this PR I just make sure not to call it with `len(filter_fields) > 1`